pepdbagent/utils.py

```diff
@@ -4,6 +4,7 @@
 from typing import Tuple, Union
 import ubiquerg
 from .exceptions import RegistryPathError
+from peppy.const import SAMPLE_RAW_DICT_KEY
```

Comment:
Locally, my intellisense doesn't see this key.. but I might have an old version of `peppy` locally that doesn't expose this.

I see over in `peppy` that it's getting added, so I am sure it is fine.